modules/seqexec/server/src/main/scala/seqexec/server/SeqTranslate.scala

```diff
@@ -217,7 +224,7 @@ class SeqTranslate(site: Site, systems: Systems[IO], settings: TranslateSettings
              tio: Timer[IO]
   ): EngineState => Option[Stream[IO, executeEngine.EventType]] = st => {
 
-    def resumeIO(c: ObserveContext[IO], resumeCmd: IO[ObserveCommandResult]): IO[Result[IO]] =
+    def resumeIO(c: ObserveContext[IO], resumeCmd: IO[ObserveCommandResult]): IO[Stream[IO, Result[IO]]] =
```

Comment:
This is a slightly weird type. You can wrap it with `Stream.eval(...).flatten` and get `Stream[IO, Result[IO]]` instead.

Reply:
I'll revisit this. I was also a bit worried about this type, but it got drowned on the large set of issues being handled